### ISSUE TITLE
chat-cli: can't post to chats

### ISSUE THREAD

**Issue by tinnus-napbus** [urbit/urbit#4758]
**Describe the bug**
when I do `;view ~ship/chat` the prompt doesn't change (I believe it should change to the chat or glyph in question?) and if I post a message it says:

```
poke failed from %chat-cli on wire /cli-command/out-message
/lib/push-hook/hoon:<[483 5].[497 45]>
"forwarding failed during transform. mark: %graph-update-1 resource: [entity=~zod name=%$]"
/lib/push-hook/hoon:<[482 5].[497 45]>
/lib/push-hook/hoon:<[471 5].[497 45]>
/lib/push-hook/hoon:<[470 5].[497 45]>
/lib/push-hook/hoon:<[469 5].[497 45]>
/lib/push-hook/hoon:<[467 5].[497 45]>
/lib/push-hook/hoon:<[466 5].[497 45]>
/lib/push-hook/hoon:<[464 5].[497 45]>
/lib/push-hook/hoon:<[462 5].[497 45]>
/lib/push-hook/hoon:<[461 5].[497 45]>
/lib/push-hook/hoon:<[460 5].[497 45]>
/lib/push-hook/hoon:<[248 9].[248 38]>
/lib/push-hook/hoon:<[247 9].[248 38]>
/lib/push-hook/hoon:<[246 7].[251 19]>
/lib/push-hook/hoon:<[240 7].[251 19]>
/lib/push-hook/hoon:<[239 7].[251 19]>
/sys/vane/gall/hoon:<[1.370 9].[1.370 37]>
~zod:chat-cli/ 
```

**To Reproduce**
Steps to reproduce the behaviour:
1. switch to chat-cli
2. ;view some chat
3. try post message

**Expected behaviour**
prompt should change to current chat and should be able to post

**System (please supply the following information, if relevant):**
 - OS: debian
 - Vere and Urbit OS versions 1.3 / u717o
 - Your ship's `%base` hash u717o

**Comment by Fang-** [comment]
Yeah I've noticed that `;view` doesn't actually update the target audience. You'll need to manually `;~ship/chat` (or use the bound glyph) afterwards. The error message you're getting says you're trying to send to `~zod/`, which doesn't exist, and so will fail.

The `;view` setting audience thing is an easy enough fix, I should be able to get that in for next ota.

**Comment by tinnus-napbus** [comment]
ohh nice I didn't realise `;[glyph]` / `;~ship/chat` was a valid command. Since you can seemingly view multiple chats simultaneously it might make sense to keep input and view separate, idk. Maybe `;help` should just tell you how to set target
